safebrowser.go

```diff
@@ -358,6 +359,12 @@ func (sb *SafeBrowser) Status() (Stats, error) {
 	return stats, sb.db.Status()
 }
 
+// WaitUntilReady blocks until the database is not in an error state,
+// or until the provided Context is cancelled.
```

Comment:
What should the expected behavior be if someone calls SafeBrowser.Close when there is a pending WaitUntilReady call?

Doing so on a TCP connection causes pending reads to unblock. I would expect similar behavior here.